### DISCUSSION TITLE
XPRA menu - move or relocate, and display resizing

### DISCUSSION THREAD

**Discussion by markknit** [Xpra-org/xpra#4431] [Q&A] [answered]
I'm using XPRA 4.3.4 on a large group of offline education servers running Ubuntu 18.04 with Nginx to host desktop applications through the browser. The setup uses an XPRA proxy server to launch the applications. However, when it's loaded there is a small menu bar in the upper left corner that interferes with access to the application controls. The users are very non-technical so getting them to move it each time won't work. Is there a way to automatically move it to another part of the display screen, or to remove it completely? I tried all of the menu related commands I could find, but none has worked. 

I am also having trouble getting the display screen to scale properly to fit the browser window. I tried the scaling commands from the proxy server side, as well as the client side, but none seem to make any change. What is the best way to do this?

Thanks!

(BTW: I will eventually upgrade the systems to Ubuntu 24.04, but that will take a while due to the number of systems, so I need to stick with this level for now.)

**Comment by totaam** [comment, marked as answer]
> I'm using XPRA 4.3.4

This is not a supported version:
https://github.com/Xpra-org/xpra/wiki/Versions

> with Nginx to host desktop applications

Please share your nginx config.

> there is a small menu bar in the upper left corner

In "Advanced options", there is an option for the "floating menu".

> I tried all of the menu related commands I could find

Such as?
I don't know what "menu related commands" would be.

> I am also having trouble getting the display screen to scale properly to fit the browser window

Don't use Debian or Ubuntu, this disability is well documented.

**Comment by markknit** [comment]
Thanks for responding quickly. Answers are below.

Mark Knittel
+1-360-441-2126
(Mobile & WhatsApp)

**Comment by totaam** [comment]
> Here is the menu I’m referring to.

Where?
Emails are stripped of their attachments, please use the github UI instead.

> I’m not seeing that in the man pages for the website

There is no man page for the html5 client.

> --tray=, --global-menus=, etc.

`global-menu` no longer exists, and it is unrelated anyway.

> So I should assume that there is no way to scale the display on Ubuntu? If I change the display manager would that help?

This is not about scaling, this is about server-side virtualization of displays.
There should be no display manager involved - xpra is the window manager on the server-side.

**Comment by markknit** [comment]
I found the floating_menu option. That worked, and I addressed the display issue. 

Thanks!
